rmm/monitor/src/rmi/rtt.rs

```
@@ -148,15 +170,14 @@ pub fn set_event_handler(mainloop: &mut Mainloop) {
         Ok(())
     });
 
-    listen!(mainloop, rmi::DATA_DESTORY, |arg, _ret, rmm| {
+    listen!(mainloop, rmi::DATA_DESTROY, |arg, _ret, rmm| {
         // rd granule lock
         let rd_granule = get_granule_if!(arg[0], GranuleState::RD)?;
         let rd = rd_granule.content::<Rd>();
         let realm_id = rd.id();
         let ipa = arg[1];
 
-        // TODO: Fix to get PA by rtt_walk
-        let pa = rmm.rmi.unmap(realm_id, ipa, GRANULE_SIZE)?;
+        let pa = rmm.rmi.data_destroy(realm_id, ipa)?;
 
         // data granule lock and change state
         let _ = set_state_and_get_granule!(pa, GranuleState::Delegated);
```

Sorry, but could you change this code?
It seems my mistake from the PR of granule transaction. There is no reason to ignore an error.
```
// before
let _ = set_state_and_get_granule!(...);
Ok(())

// after
set_state_and_get_granule!(...)
```

---

Reflected like the below, as `set_state_and_get_granule`  returns () type.
```
set_state_and_get_granule!(...)?
Ok(())
```